apps/src/templates/instructions.jsx

```diff
@@ -0,0 +1,59 @@
+var msg = require('../locale');
+
+module.exports = React.createClass({
+
+  propTypes: {
+    puzzleTitle: React.PropTypes.string,
+    instructions: React.PropTypes.string,
+    instructions2: React.PropTypes.string,
+    renderedMarkdown: React.PropTypes.string,
+    authoredHints: React.PropTypes.array,
+    markdownClassicMargins: React.PropTypes.bool,
+    aniGifURL: React.PropTypes.string
```

Comment:
Are any of these required? If so, we can tack on .isRequired, and get warnings if someone doesnt set them